models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -478,7 +480,8 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
         param[which(param[, 1] == "plantWoodInit"), 2] <- PEcAn.utils::ud_convert(IC.pools$wood, "g m-2", "g m-2")
       }
       ## laiInit m2/m2
-      lai <- try(ncdf4::ncvar_get(IC.nc,"LAI"),silent = TRUE)
+      #lai <- try(ncdf4::ncvar_get(IC.nc,"LAI"),silent = TRUE)
+      lai <- IC.pools$LAI
```

Comment:
Any reason not to delete the commented line?

Reply:
No as those lines are identical I will delete the commented out line, must've been left over from debugging.